android/src/main/java/com/swmansion/rnscreens/ScreenStack.kt

```diff
@@ -335,8 +336,9 @@ class ScreenStack(context: Context?) : ScreenContainer(context) {
             fragmentWrapper.screen.stackPresentation === Screen.StackPresentation.TRANSPARENT_MODAL
 
         private fun needsDrawReordering(fragmentWrapper: ScreenFragmentWrapper): Boolean =
-            fragmentWrapper.screen.stackAnimation === StackAnimation.SLIDE_FROM_BOTTOM ||
+            Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU ||
```

Comment:
Can you add a comment here why we're comparing Android version (just for a context, that since API 33 the default animation has changed and because of the "sliding" we need to draw reordering 👍)